src/BenchmarkDotNet/Portability/Cpu/HardwareIntrinsics.cs

```diff
@@ -117,6 +117,13 @@ static IEnumerable<string> GetCurrentProcessInstructionSets(Platform platform)
             GetIsSupported("System.Runtime.Intrinsics.X86.Sse3");
 #endif
 
+        internal static bool IsX86Ssse3Supported =>
+#if NET6_0_OR_GREATER
+            Ssse3.IsSupported;
+#elif NETSTANDARD
+            GetIsSupported("System.Runtime.Intrinsics.X86.Ssse3");
+#endif
+
```

Comment:
Does the `#if` here really help with performance in a noticeable way? Changing all of those to just this would make everything much shorter:
```suggestion
        internal static bool IsX86Ssse3Supported { get; } =
            GetIsSupported("System.Runtime.Intrinsics.X86.Ssse3");
```

Reply:
> Does the #if here really help with performance in a noticeable way? Changing all of those to just this would make everything much shorter:

This code is going to be executed by NativeAOT which would require some extra ceremony to get reflection working.